metrics/density/docker_memory_usage.sh

```diff
@@ -251,6 +252,10 @@ EOF
 			die "Failed to find PSS for $HYPERVISOR_PATH"
 		fi
 
+		virtiofsd_mem="$(get_pss_memory "$VIRTIOFSD_PATH")"
+		if [ "$virtiofsd_mem" == "0" ]; then
+			echo "Failed to find PSS for $VIRTIOFSD_PATH"
```

Comment:
Should this be `warn "Failed to find PSS for $VIRTIOFSD_PATH"`, or `die`?

Reply:
should add "Warning:" as the first printed word?

Reply:
like the other place, you can use [`warning()`](https://github.com/kata-containers/tests/blob/4a46b74d542f48c0d34c3a5da82333469515f455/metrics/lib/common.bash#L39-L45) from the common library code, which prepends the message with 'WARNING: ' for you.